src/scripts/main.js

```diff
@@ -2,7 +2,18 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+for (const every of people) {
+  const table = document.createElement('tr');
+
+  table.insertAdjacentHTML('beforeend', `
+  <td>${every.name}</td>
+  <td>${every.sex === 'm' ? 'Male' : 'Female'}</td>
+  <td>${every.born}</td>
+  <td>${every.died}</td>
+  <td>${every.died - every.born}</td>
+  <td>${Math.ceil(every.died / 100)}</td>
+  `);
```

Comment:
Add small formatting here:
```suggestion
  table.insertAdjacentHTML('beforeend', `
    <td>${every.name}</td>
    <td>${every.sex === 'm' ? 'Male' : 'Female'}</td>
    <td>${every.born}</td>
    <td>${every.died}</td>
    <td>${every.died - every.born}</td>
    <td>${Math.ceil(every.died / 100)}</td>
  `);
```